backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
+          type: boolean
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        sex:
+          $ref: "#/components/schemas/ontology_element_array"
+        ethnicity:
+          $ref: "#/components/schemas/ontology_element_array"
+        development_stage:
+          $ref: "#/components/schemas/ontology_element_array"
+        dataset_deployments:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                type: string
+        linked_genesets:
+          type: array
+          items:
+            type: string
+        cell_type:
+          $ref: "#/components/schemas/ontology_element_array"
+        cell_count:
+          type: integer
+        X_approximate_distribution:
+          $ref: "#/components/schemas/distribution"
+        schema_version:
+          type: string
+    dataset_preview:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - tissue
+        - assay
+        - disease
+        - organism
+        - tombstone
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tombstone:
+          type: boolean
+    dataset_asset:
+      type: object
+      properties:
+        id:
+          type: string
+        dataset_id:
+          type: string
+        filetype:
+          type: string
+          enum: [H5AD, RDS, CXG]
+        filename:
+          type: string
+        type:
+          type: string
+          enum: [REMIX, ORIGINAL]
+        s3_uri:
+          type: string
+    gene_set:
+      type: object
+      properties:
+        name:
+          type: string
+        id":
+          type: string
+        description:
+          type: string
+        linked_datasets:
+          type: array
+          items:
+            type: string
+    ontology_element:
+      type: object
+      properties:
+        ontology_term_id:
+          type: string
+        label:
+          type: string
+    ontology_element_array:
+      type: array
+      items:
+        $ref: "#/components/schemas/ontology_element"
+    problem:
+      type: object
+      description: Error message container for HTTP APIs.
+      properties:
+        type:
+          type: string
```

Comment:
Should this be an enum?

Reply:
It could be, but it doesn't really matter too much. Custom descriptor from the BE.